test/unit/winston/create-logger.test.js

```diff
@@ -0,0 +1,107 @@
+const winston = require("../../../lib/winston");
+const assume = require("assume");
+const isStream = require("is-stream");
+const {format} = require("../../../lib/winston");
+const TransportStream = require("winston-transport");
+
+describe('Create Logger', function () {
```

Comment:
These tests were originally a part of the `logger.test.js` file. I chose to break them out as they're intended to test the `create-logger.js` file (even if they're currently opperating on the `winston.js` file. Hence this is more of an integration test instead of a unit test but it's a start IMO.